src/main/java/seedu/address/model/issue/IssueList.java

```diff
@@ -99,6 +100,17 @@ public void setIssues(List<Issue> issues) {
         FXCollections.sort(internalList);
     }
 
+    /**
+     * Checks if any issues have the given room associated with it
+     *
+     * @param target Room to check if it has issues associated with it.
+     * @return true if there are issues with the given room associated with it
```

Comment:
Is this missing a fullstop, and should it start with a capital letter?

Saw this in `Model.java` and `AddressBook.java` too.

Reply:
Yes, fixed.